app/controllers/api/v1/notes_controller.rb

```diff
@@ -9,8 +9,7 @@ def index
   end
 
   def create
-    note = current_user.notes.new(note_params)
-    note.save!
+    note = current_user.notes.create!(note_params)
```

Comment:
Do we need this variable called `note`? Is it being used within the scope? If not let's remove it. Raise a separate PR for this.

Reply:
Yes, I had not noticed it. I will raise a separate PR for this.